README.md

```diff
@@ -50,3 +50,14 @@ npm run build:ds
 npm run build
 ```
 
+## Architecture decision records
+
+### Create a new architecture decision record
+
+- Install [adr tools](https://github.com/adr/adr-tools/blob/patch-1/INSTALL.md)
```

Comment:
This repository has been archived in 2022. I think we should probably not use this tool.

Reply:
@fmvilas most of adr tools I find in the OSS are outdated I'm going to build a small one that implement this template.

Reply:
I think a manual approach is fine. It's a really simple template :)